.github/workflows/hatch_ci.yml

```diff
@@ -148,3 +173,66 @@ jobs:
           ls dist/*.tar.gz | xargs -n1 hatch run python -m twine check
           ls dist/*.whl | xargs -n1 hatch run python -m twine check
         shell: bash
+
+      - name: Run Whitesource Scan
+        if: ${{ 
+          github.repository_owner == 'SolaceDev' && 
+          inputs.whitesource_product_name != '' && 
+          inputs.whitesource_project_name != '' 
+          }}
+        uses: SolaceDev/solace-public-workflows/.github/actions/whitesource-scan@main
+        with:
+          whitesource_product_name: ${{ inputs.whitesource_product_name }}
+          whitesource_project_name: ${{ inputs.whitesource_project_name }}
+          whitesource_api_key: ${{ secrets.WHITESOURCE_API_KEY }}
+
+      - name: Run WhiteSource Policy Gate
+        if: ${{ 
+          github.repository_owner == 'SolaceDev' && 
```

Comment:
- Do not run if product name is project name is not specified (We cannot map the results into Mend backend)
- Only run in SolaceDev and `main` branch. Once we make the cutover to SolaceLabs, this can be easily changed.